Index.html

```diff
@@ -5,11 +5,190 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+
+    <link rel="stylesheet" href="/css/Main.css">
```

Comment:
Sadly, your CSS is not being displayed with your HTML 😥 There is a path issue most likely that is interfering with the desired functionality. Please make sure that your website is displaying properly so we can review your design with Figma's 😁

![Captura](https://user-images.githubusercontent.com/15807343/148427103-0477ef4c-d3f7-40ba-a713-23c49db3fdf6.PNG)